ajax/libs/bootstrap-material-datetimepicker/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bootstrap-material-datetimepicker",
+  "description": "Datepicker for bootstrap-material",
+  "version": "2.5.3",
+  "keywords": [
+    "css",
+    "js",
+    "mobile-first",
+    "responsive",
+    "front-end",
+    "web"
+  ],
+  "filename": "js/bootstrap-material-datetimepicker.min.js",
+  "dependencies": {
+    "moment": "~2.10.2",
+    "bootstrap": "~3.3.4",
+    "jquery": "1.11.0",
+    "bootstrap-material-design": "~0.3.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/T00rk/bootstrap-material-datetimepicker"
+  },
+  "author": "T00rk",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/T00rk/bootstrap-material-datetimepicker.git",
+    "basePath": "",
+    "files": [
+      "+(js|css)/bootstrap-material-datetimepicker*.+(css|js|map)",
+      "font/Material-Design-Icons*.+(eot|svg|ttf|woff|woff2)"
```

Comment:
fonts didn't been referenced in the js and css files, and we have: https://cdnjs.com/libraries/material-design-icons, so no need to add that.